ioctl/cmd/account/accountcreate.go

```diff
@@ -84,7 +85,9 @@ func accountCreate() error {
 		if addr == nil {
 			return output.NewError(output.ConvertError, "failed to convert public key into address", nil)
 		}
+		//ethAddr, _ := address.FromString(addr.String())
 		newAccount := generatedAccount{
+			EthAddress: addr.Hex(),
```

Comment:
this only shows when a new account is created, which is not frequently used, `account createadd` is used more and should reflect this change.

we should also add this to `ioctl account info` command